steamcmd_servers/neosvr/egg-neos-vr-headless-client.json

```diff
@@ -0,0 +1,59 @@
+{
+    "_comment": "Egg for neos vr headless client/server",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-06-08T09:15:41-04:00",
+    "name": "NeosVR Headless Client",
+    "author": "synk@nasulex.net",
+    "description": "Headless client for NeosVR.",
+    "features": null,
+    "images": [
+        "nasulex\/neos:latest"
```

Comment:
Why does this require a custom image? Are you sure the existing images are not enough? If so, you should PR the changes to the[ image repository.](https://github.com/parkervcp/images)

Reply:
It seems like I would need my own image because it crashes on the mono core one so I guess ill do that then + the entry point doesnt run steamcmd so I can have it do that too